### DISCUSSION TITLE
An issue related to the education pacl

### DISCUSSION THREAD

**Discussion by Andy-XO** [community/community#81627] [GitHub Education]
### Hi everyone,

Hello ,
Atharva this side.
I was using my ID card to activate the Github education pack.
but it is showing that the Id card is alreday used.I did'nt use it. How's that even possible.
kindly guide me on this issue. tell me who is using my id card or at leat my account status.
please help me fo it.

**Comment by anuragjain-git** [comment]
1. Review GitHub Education Terms
2. Visit : https://support.github.com/features/education
3. Provide Proof
